### DISCUSSION TITLE
ERROR with BT-736-Lot, BT-743-Lot and BT-764-Lot

### DISCUSSION THREAD

**Discussion by user6330** [OP-TED/eForms-SDK#582] [Q&A] [answered]
Same case as #580 

![image](https://github.com/OP-TED/eForms-SDK/assets/127392893/fa36205c-10cb-4bac-a0e5-739f4284f75a)

![image](https://github.com/OP-TED/eForms-SDK/assets/127392893/eae2165f-a8a2-4373-9c1f-e2687dd6b432)

**Comment by YvesJo** [comment, marked as answer]
Hi,
This is exactly the same kind of issue you highlighted in #580 , so I close it as duplicate
